molecule/default/molecule.yml

```diff
@@ -5,9 +5,6 @@ dependency:
     requirements-file: molecule/default/requirements.yml
 driver:
   name: docker
-lint: |
```

Comment:
@RanabirChakraborty I'm not sure you meant to commit this.

Reply:
Here we are using ansible 2.13.5 which is using ansible-lint 6.8.2 (which is the latest), in this new version they have included various syntax checks, and our repo files are not aligned with those guidelines and this is what failing in local as well as in CI. Below are the errors.
```
ansible-lint 6.8.2 using ansible 2.13.5
  INFO     Running ansible-galaxy collection install -vr requirements.yml -p /home/runner/.cache/ansible-compat/d41849/collections
  INFO     Set ANSIBLE_LIBRARY=/home/runner/.cache/ansible-compat/d41849/modules:/home/runner/.ansible/plugins/modules:/usr/share/ansible/plugins/modules
  INFO     Set ANSIBLE_COLLECTIONS_PATH=/home/runner/.cache/ansible-compat/d41849/collections:/home/runner/.ansible/collections:/usr/share/ansible/collections
  INFO     Set ANSIBLE_ROLES_PATH=/home/runner/.cache/ansible-compat/d41849/roles:roles:/home/runner/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles
  INFO     Running ansible-galaxy collection install -v --force -p /home/runner/.cache/ansible-compat/d41849/collections .
  INFO     Set ANSIBLE_LIBRARY=/home/runner/.cache/ansible-compat/d41849/modules:/home/runner/.ansible/plugins/modules:/usr/share/ansible/plugins/modules
  INFO     Set ANSIBLE_COLLECTIONS_PATH=/home/runner/.cache/ansible-compat/d41849/collections:/home/runner/.ansible/collections:/usr/share/ansible/collections
  INFO     Set ANSIBLE_ROLES_PATH=/home/runner/.cache/ansible-compat/d41849/roles:roles:/home/runner/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles
  INFO     Running ansible-galaxy collection install -v --force -p /home/runner/.cache/ansible-compat/d41849/collections .
  INFO     Discovered files to lint using: git ls-files --cached --others --exclude-standard -z
  INFO     Excluded removed files using: git ls-files --deleted -z
  INFO     Discovered files to lint using: git ls-files --cached --others --exclude-standard -z
  INFO     Excluded removed files using: git ls-files --deleted -z
  INFO     Executing syntax check on playbooks/playbook_verify.yml (0.58s)
  INFO     Executing syntax check on playbooks/playbook.yml (0.62s)
  WARNING  Listing 111 violation(s) that are fatal
  playbooks/playbook.yml:12: risky-file-permissions: File permissions unset or incorrect. (warning)
  playbooks/playbook.yml:12: task_has_valid_name: Invalid task name 'Copying the setup file'
  roles/wildfly_driver/defaults/main.yml:2: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_driver/defaults/main.yml:3: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_driver/defaults/main.yml:4: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_driver/defaults/main.yml:5: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_driver/defaults/main.yml:6: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_driver/defaults/main.yml:7: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_driver/defaults/main.yml:8: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_driver/defaults/main.yml:9: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_driver/defaults/main.yml:10: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_driver/defaults/main.yml:11: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_driver/meta/main.yml:1: schema[meta]: {'name': 'EL', 'versions': [8]} is not valid under any of the given schemas (warning)
  roles/wildfly_install/defaults/main.yml:2: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_install/defaults/main.yml:3: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_install/defaults/main.yml:4: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_install/defaults/main.yml:6: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_install/defaults/main.yml:8: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_install/defaults/main.yml:9: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_install/defaults/main.yml:10: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_install/defaults/main.yml:11: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_install/defaults/main.yml:12: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_install/defaults/main.yml:13: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_install/defaults/main.yml:16: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
  roles/wildfly_install/defaults/main.yml:17: role_vars_start_with_role_name: All role parameters must be named ^<role_name>_.* (warning)
```

Reply:
Normally, the [.ansible-lint](https://github.com/ansible-middleware/wildfly/blob/main/.ansible-lint#L23) file in the project should make those errors, just warning and not break the build. So, you don't need to remove ansible-lint.

Those two errors, however, need to be fixed up:

```
playbooks/playbook.yml:12: risky-file-permissions: File permissions unset or incorrect. (warning)
playbooks/playbook.yml:12: task_has_valid_name: Invalid task name 'Copying the setup file'
```

Reply:
I have fixed all the possible errors and ansible-lint back but still, the CI is failing. Not understanding the failure ERROR.